arxiv/ops/fastly_log_ingest/app.py

```diff
@@ -61,7 +61,10 @@ def _status_to_severity(self, status: int):
     def __init__(self, message: Message, arrival_time: float):
         self.message = message
         self.arrival_time = arrival_time
-        self.data: dict = json.loads(message.data.decode('utf-8').strip())
+        try:
+            self.data: dict = json.loads(message.data.decode('utf-8').strip())
+        except:
+            logger.warning(f'Failed to log the following message: {message.data.decode("utf-8")}')
```

Comment:
Since the decode could have been the problem, use decode('utf8', errors='replace') in the error message.

Please log the exception message too.